src/gen/js/genReduxActions.ts

```diff
@@ -48,14 +48,17 @@ function renderReduxActionBlock(spec: ApiSpec, op: ApiOperation, options: Client
     if (required.length) params += ', options'
     else params = 'options'
   }
-
+  const docsArray=renderOperationDocs(op)
```

Comment:
Ohhh yes you are right, didn't tough about that as mostly i only wanted to get completion in action's arguments.

So probably this needs more work, can't just use the same gen docs from the other methods.